### PR TITLE
DX-346 accounts add docs for extending accounts ep with new cdr attributes

### PR DESCRIPTION
**Pull request by basiq-ash**
BASIQ Schema mapped with OB Schema for accounts MetaData Extensions Docs

- fees
- depositRates
- lendingRates
- loan
- creditCard

### CODE REVIEW

**Review by basiq-brad — APPROVED** [review]
LGTM

File: data.yml [hunk 1]
```diff
@@ -2043,6 +2057,26 @@ components:
           enum:
           - available
           - unavailable
+        fees:
+          type: array
+          items:
+            $ref: '#/components/schemas/feesaccountMetadata'
```
Comment (basiq-jon) [inline data.yml:2063]: feesAccountMetadata?

File: data.yml [hunk 2]
```diff
@@ -2043,6 +2057,26 @@ components:
...
+          items: 
+            $ref: '#/components/schemas/depositRateaccountMetadata'
+        lendingRates:
+          type: array
+          items: 
+            $ref: '#/components/schemas/lendingRateaccountMetadata'
+        loan:
+          type: array
+          items: 
+            $ref: '#/components/schemas/loanaccountMetadata'
```
Comment (basiq-jon) [inline data.yml:2075]: loanAccountMetadata?

File: data.yml [hunk 3]
```diff
@@ -2043,6 +2057,26 @@ components:
...
+        creditCard:
+          type: array
+          items: 
+            $ref: '#/components/schemas/creditCardaccountMetadata'
```
Comment (basiq-jon) [inline data.yml:2079]: creditCardAccountMetadata?

File: data.yml [hunk 4]
```diff
@@ -2054,6 +2088,144 @@ components:
...
+          type: string
+          description: >- 
+            (mandatory) - Date that the next payment for the card is due. 
+    loanaccountMetadata: 
+      type: object
+      properties: 
+        startDate: 
+          type: string
+          description: >- 
+            (optional) - start date for the loan.
```
Comment (basiq-jon) [inline data.yml:2120]: Start date

File: data.yml [hunk 5]
```diff
@@ -2043,6 +2057,26 @@ components:
...
           - available
           - unavailable
+        fees:
+          type: array
+          items:
+            $ref: '#/components/schemas/feesaccountMetadata'
+        depositRates:
+          type: array
+          items: 
+            $ref: '#/components/schemas/depositRateaccountMetadata'
```
Comment (basiq-jon) [inline data.yml:2067]: depositRateAccountMetadata?

File: data.yml [hunk 6]
```diff
@@ -2043,6 +2057,26 @@ components:
...
+        lendingRates:
+          type: array
+          items: 
+            $ref: '#/components/schemas/lendingRateaccountMetadata'
```
Comment (basiq-jon) [inline data.yml:2071]: lendingRateAccountMetadata?

File: data.yml [hunk 7]
```diff
@@ -2054,6 +2088,144 @@ components:
...
+            (optional) - The period after which the calculated amount(s) are 'applied' (i.e. debited or credited) to the account.
+    feesaccountMetadata: 
+      required:
+      - name
+      - feeType
+      type: object
+      properties:
+        name:
+          type: string
+          description: (mandatory Field) Name of the fee
```
Comment (basiq-jon) [inline data.yml:2206]: Inconsistent use of (mandatory) and (mandatory Field). Also some have `-` and some don't.

File: data.yml [hunk 8]
```diff
@@ -2054,6 +2088,144 @@ components:
...
+          - PERIODIC
+          - PURCHASE
+          - TRANSACTION
+          - UPFRONT
+          - VARIABLE
+          - WITHDRAWAL
+        amount:
+          type: string
+          description: >-
+            (conditional) The amount charged for the fee.
```
Comment (basiq-jon) [inline data.yml:2224]: Like this one, inconsistent - others have a dash.

**Comment by basiq-ash** [comment]
Resolved as a part of https://github.com/basiqio/api-ref/pull/222

**Pull request merged**